hyundai_canfd.dbc

```diff
@@ -84,6 +84,7 @@ BO_ 101 BRAKE: 32 XXX
  SG_ CHECKSUM : 0|16@1+ (1,0) [0|65535] "" XXX
  SG_ COUNTER : 16|8@1+ (1,0) [0|255] "" XXX
  SG_ BRAKE_PRESSED : 57|1@1+ (1,0) [0|3] "" XXX
+ SG_ BRAKE_PRESSURE : 40|16@1+ (1,0) [0|695] "" XXX
```

Comment:
Only used 10 bits at max force/position, but the rest were zeros, so map the full signal?